tools/nodeset_compiler/backend_open62541_datatypes.py

```diff
@@ -99,6 +99,8 @@ def generateNodeIdCode(value):
     elif value.s != None:
         v = makeCLiteral(value.s)
         return u"UA_NODEID_STRING(ns[%s], \"%s\")" % (value.ns, v)
+    elif value.g != None:
+        return u"UA_NODEID_GUID(ns[%s], %s)" % (value.ns, generateGuidCode(value.gAsStrings()))
     raise Exception(str(value) + " no NodeID generation for bytestring and guid..")
```

Comment:
the error string in the exception still needs to be adjusted?

Reply:
You're right. But that's not all. See above. My python skills are bad.